.circleci/config.yml

```diff
@@ -70,6 +70,11 @@ utils:
     command: |
       make secret-scan
 
+  run_security_scan: &run_security_scan
+    name: Run security scan
+    command: |
+      make security-scan
```

Comment:
```suggestion
    command: make security-scan
```

Reply:
Should I change all the other instances of it (for secret-scan and pylint) too?

Reply:
Yeah if you want to 🙂
it's super minor, just saw it in diff, hence the comment)

Reply:
Done :)